src/plugins/requestAuthPlugin.test.js

```diff
@@ -56,65 +95,72 @@ const MOCK_AUTHED_REQUEST = {
 	authorize: () => Observable.of(MOCK_AUTHED_REQUEST),
 };
 
-const GOOD_MOCK_FETCH_RESULT = Promise.resolve({
-	text: () => Promise.resolve('{}')
-});
+const GOOD_MOCK_FETCH_RESULT = Promise.resolve({ text: () => Promise.resolve('{}') });
 const BAD_MOCK_FETCH_RESULT = Promise.resolve({ text: () => Promise.resolve(undefined) });
 
-const OAUTH_AUTH_URL = 'http://example.com/auth_fakeout';
-const OAUTH_ACCESS_URL = 'http://example.com/access_fakeout';
-const MOCK_CODE = { grant_type: 'anonymous_code', token: 'mock_anon_code' };
-
 describe('getAnonymousCode$', () => {
-	it('sets OAUTH_AUTH_URL as opts.url', function(done) {
+	it('sets MOCK_OAUTH.auth_url as opts.url', function(done) {
 		// given the config, the mocked request function should return the auth url in code.url
 		spyOn(global, 'fetch').and.callFake((url, opts) => {
-			expect(url.startsWith(OAUTH_AUTH_URL)).toBe(true);
+			expect(url.startsWith(MOCK_OAUTH.auth_url)).toBe(true);
 			return GOOD_MOCK_FETCH_RESULT;
 		});
-		getAnonymousCode$({ oauth, OAUTH_AUTH_URL }).subscribe(done);
+
+		getAnonymousCode$({ ...MOCK_SERVER_APP }, null).subscribe(done);
 	});
+
 	it('throws error when response cannot be JSON parsed', function() {
 		spyOn(global, 'fetch').and.callFake((url, opts) => BAD_MOCK_FETCH_RESULT);
 
-		return getAnonymousCode$({ oauth, OAUTH_AUTH_URL })
+		return getAnonymousCode$({ ...MOCK_SERVER_APP }, null)
 			.toPromise()
 			.catch(err => {
 				expect(err).toEqual(jasmine.any(Error));
 			});
 	});
-
 });
 
 describe('getAccessToken$', () => {
-	it('sets OAUTH_ACCESS_URL as opts.url', function(done) {
+	it('sets MOCK_OAUTH.access_url as opts.url', function(done) {
 		spyOn(global, 'fetch').and.callFake((url, opts) => {
-			expect(url.startsWith(OAUTH_ACCESS_URL)).toBe(true);
+			expect(url.startsWith(MOCK_OAUTH.access_url)).toBe(true);
 			return GOOD_MOCK_FETCH_RESULT;
 		});
-		const getToken$ = getAccessToken$({ oauth, OAUTH_ACCESS_URL }, null);
+
+		const getToken$ = getAccessToken$({ ...MOCK_SERVER_APP }, null);
 		getToken$(MOCK_HEADERS)(MOCK_CODE).subscribe(done);
 	});
-	it('throws an error when no oauth.key is supplied', function() {
-		const oauthNoKey = { ...oauth };
-		delete oauthNoKey.key;
-		expect(() => getAccessToken$({ oauth: oauthNoKey, OAUTH_ACCESS_URL }, null))
+
+	xit('throws an error when no oauth.key is supplied', function() {
```

Comment:
This test (and the one after) is causing `MOCK_SERVER_APP` to be mutated, but I'm not sure why this is happening. 

Because it's deleting `MOCK_SERVER_APP.oauth.key`, the [subsequent test](https://github.com/meetup/meetup-web-platform/pull/264/files#diff-b061c55b4fb48f7b0f38751009857f4cR150) is failing. 

Anyone have thoughts? cc @mmcgahan